activiti-cloud-services-query/activiti-cloud-services-query-rest/src/main/java/org/activiti/cloud/services/query/events/handlers/TaskCandidateGroupRemovedEventHandler.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright 2018 Alfresco, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.activiti.cloud.services.query.events.handlers;
+
+import org.activiti.cloud.services.api.events.ProcessEngineEvent;
+import org.activiti.cloud.services.query.app.repository.TaskCandidateGroupRepository;
+import org.activiti.cloud.services.query.events.TaskCandidateGroupRemovedEvent;
+import org.activiti.cloud.services.query.model.TaskCandidateGroup;
+import org.activiti.engine.ActivitiException;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import javax.persistence.EntityManager;
+
+@Component
+public class TaskCandidateGroupRemovedEventHandler implements QueryEventHandler {
+
+    private final TaskCandidateGroupRepository taskCandidateGroupRepository;
+    private final EntityManager entityManager;
+
+    @Autowired
+    public TaskCandidateGroupRemovedEventHandler(TaskCandidateGroupRepository taskCandidateGroupRepository, EntityManager entityManager) {
+        this.taskCandidateGroupRepository = taskCandidateGroupRepository;
+        this.entityManager = entityManager;
+    }
+
+    @Override
+    public void handle(ProcessEngineEvent event) {
+        TaskCandidateGroupRemovedEvent taskCandidateGroupRemovedEvent = (TaskCandidateGroupRemovedEvent) event;
+        TaskCandidateGroup taskCandidateGroup = taskCandidateGroupRemovedEvent.getTaskCandidateGroup();
+
+        // remove from database
+        try {
+            taskCandidateGroupRepository.delete(taskCandidateGroup);
+        } catch(Exception cause) {
```

Comment:
Catch `Throwable`.

Reply:
I think Exception is more appropriate in this case